main/php_syslog.c

```diff
@@ -37,6 +38,8 @@
 #define syslog std_syslog
 #endif
 
+zend_bool have_called_openlog = 0;
```

Comment:
This is not thread safe, thus it should go into globals.

Reply:
> This is not thread safe, thus it should go into globals.

Not sure I understand.  What's an example of a global variable that I can use as an example?

Reply:
Yeah, something in this direction. Perhaps core globals were more suitable, as it's not something SAPI specific. Another option in PHP 7 is using ZEND_TLS, but should check whether its usage were suitable in this case.

Reply:
Isn't core globals for configuration state?

Reply:
> This is not thread safe, thus it should go into globals.

@weltling Done